### PR TITLE
Change Notification Team/Channel Nodejs

### PR DESCRIPTION
**Pull request by Mohammed-MSFT**
Hi @ChetanSharma-msft, Raised PR for Change Notification for team/channel please review and add your comments.

### CODE REVIEW

File: samples/graph-change-notification-team-channel-nodejs/nodejs/.env [hunk 1]
```diff
@@ -0,0 +1,9 @@
+MicrosoftAppId = "<<App ID>>"
+MicrosoftAppPassword = "<<App Secret>>"
+MicrosoftAppTenantId = ""
+AllowedHosts = "*"
+notificationUrl= ""
+BaseUrl= "<<ngrok>>"
+Base64EncodedCertificate= ""
+EncryptionCertificateId= "graph-notification"
+CertificateThumbprint= ""
```
Comment (ChetanSharma-msft) [inline samples/graph-change-notification-team-channel-nodejs/nodejs/.env:9]: Check if all of these required and also added the details in Readme file.
What exact values we need to provide here in Readme file.

File: samples/graph-change-notification-team-channel-nodejs/nodejs/README.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+description: This is a sample application which demonstrates the use of team/Channel subscription that will post notifications when user create/edit/delete team/channel.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- nodejs
+extensions: 
+ contentType: samples
+ createdDate: "25-08-2022 04:30:15"
```
Comment (ChetanSharma-msft) [inline samples/graph-change-notification-team-channel-nodejs/nodejs/README.md:12]: This date is wrong.

File: samples/graph-change-notification-team-channel-nodejs/nodejs/README.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+## Run app locally
+
+### Register your application with Azure AD
+
+1. Register a new application in the [Azure Active Directory � App Registrations](https://go.microsoft.com/fwlink/?linkid=2083908) portal.
+2. On the overview page, copy and save the **Application (client) ID, Directory (tenant) ID**. You�ll need those later when updating your Teams application manifest and in the appsettings.json.
+3. Navigate to **API Permissions**, and make sure to add the follow permissions:
+-   Select Add a permission
+-   Select Microsoft Graph -> Application permissions.
+   - `Channel.ReadBasic.All`,`ChannelSettings.Read.All`,`Directory.ReadWrite.All`,`Group.ReadWrite.All`
```
Comment (ChetanSharma-msft) [inline samples/graph-change-notification-team-channel-nodejs/nodejs/README.md:42]: Do we need all of these permissions ?

File: samples/graph-change-notification-team-channel-nodejs/nodejs/README.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+    git clone https://github.com/OfficeDev/Microsoft-Teams-Samples.git
+    ```
+
+  A) Or from Visual Studio code
+
+  - Launch Visual Studio code
+  - File -> Open Folder
+  - Navigate to `samples/graph-change-notification-team-channel/nodejs` folder
+  - Select `ChangeNotiifcation` solution file
+  - Press `F5` to run the project
```
Comment (ChetanSharma-msft) [inline samples/graph-change-notification-team-channel-nodejs/nodejs/README.md:82]: Node sample will not run using F5 I think,
Please check the steps in another sample.

We need to install the packages etc...
Refer another NodeJS sample's Readme

File: samples/graph-change-notification-team-channel-nodejs/nodejs/README.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+  B) Run ngrok - point to port 3978
+
+   ```bash
+     ngrok http -host-header=rewrite 3978
+   ```  
+
+## Instruction for appsetting
+1. Provide MicrosoftAppId, MicrosoftAppPassword and MicrosoftAppTenantId in the appsetting that is created in Azure.
+2. Provide the ngrok url as  "BaseUrl" in appsetting on which application is running on.
+3. You should be having Base64EncodedCertificate and CertificateThumbprint value from *Create and install Self-Signed certificate* step.
```
Comment (ChetanSharma-msft) [inline samples/graph-change-notification-team-channel-nodejs/nodejs/README.md:93]: Check if you have mentioned all the appsettings key's here with details to get the values.

File: samples/graph-change-notification-team-channel-nodejs/nodejs/README.md [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+
+## Instruction for manifest
+1. Fill any GUID for <APP-ID>. You can also put your MicrosoftAppId here.
```
Comment (ChetanSharma-msft) [inline samples/graph-change-notification-team-channel-nodejs/nodejs/README.md:96]: <Manifest ID>

Manifest ID

File: samples/graph-change-notification-team-channel-nodejs/nodejs/Manifest/manifest.json [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+  ],
+  "permissions": [
+    "identity",
+    "messageTeamMembers"
+  ],
+  "validDomains": [
+    "*.ngrok.io"
+  ],
+  "webApplicationInfo": {
+    "id": "<<APP ID>>",
```
Comment (ChetanSharma-msft) [inline samples/graph-change-notification-team-channel-nodejs/nodejs/Manifest/manifest.json:44]: MICROSOFT-APP-ID

File: samples/graph-change-notification-team-channel-nodejs/nodejs/.env [hunk 8]
```diff
@@ -0,0 +1,9 @@
+MicrosoftAppId = "<<App ID>>"
```
Comment (ChetanSharma-msft) [inline samples/graph-change-notification-team-channel-nodejs/nodejs/.env:1]: MICROSOFT-APP-ID

File: samples/graph-change-notification-team-channel-nodejs/nodejs/client/package.json [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+    "date-fns": "^2.29.2",
+    "moment": "^2.29.4",
+    "react": "17.0.0",
+    "react-dom": "17.0.0",
+    "react-excel-renderer": "^1.1.0",
+    "react-router-dom": "6.2.2",
+    "react-scripts": "^4.0.0",
+    "reactstrap": "^9.1.4",
+    "uuid": "^9.0.0",
+    "xlsx": "^0.18.5"
```
Comment (ChetanSharma-msft) [inline samples/graph-change-notification-team-channel-nodejs/nodejs/client/package.json:20]: Check if we are using all of these packages.

Comment (ChetanSharma-msft) [reply samples/graph-change-notification-team-channel-nodejs/nodejs/client/package.json:20]: If not, remove

File: samples/graph-change-notification-team-channel-nodejs/nodejs/client/src/components/channelnotification.jsx [hunk 10]
```diff
@@ -0,0 +1,103 @@
+// <copyright file="dashboard.jsx" company="Microsoft Corporation">
```
Comment (ChetanSharma-msft) [inline samples/graph-change-notification-team-channel-nodejs/nodejs/client/src/components/channelnotification.jsx:1]: File name is different and here it's different.
Rename it as channel-notification.jsx

File: samples/graph-change-notification-team-channel-nodejs/nodejs/client/src/components/teamnotification.jsx [hunk 11]
```diff
@@ -0,0 +1,99 @@
+// <copyright file="dashboard.jsx" company="Microsoft Corporation">
```
Comment (ChetanSharma-msft) [inline samples/graph-change-notification-team-channel-nodejs/nodejs/client/src/components/teamnotification.jsx:1]: File name is different and here it's different.
Rename it as team-notification.jsx

File: samples/graph-change-notification-team-channel-nodejs/nodejs/client/src/index.css [hunk 12]
```diff
@@ -0,0 +1,208 @@
...
+  width: 96vw;
+  display: grid;
+  grid-template-columns: auto auto auto;
+  padding: 10px;
+  color: #b7b7b7;
+  font-family: segoe ui;
+}
+
+.chat-window-sidepanel {
+  height: 80vh;
```
Comment (ChetanSharma-msft) [inline samples/graph-change-notification-team-channel-nodejs/nodejs/client/src/index.css:43]: Check if we are using all of these CSS

Comment (ChetanSharma-msft) [reply samples/graph-change-notification-team-channel-nodejs/nodejs/client/src/index.css:43]: If not, remove.

File: samples/graph-change-notification-team-channel-nodejs/nodejs/client/src/models/dashboard-state.js [hunk 13]
```diff
@@ -0,0 +1,7 @@
+const DashboardState = {
+    Default: 'default',
+    View: 'view',
+    Edit: 'edit',
+}
```
Comment (ChetanSharma-msft) [inline samples/graph-change-notification-team-channel-nodejs/nodejs/client/src/models/dashboard-state.js:5]: Why do we need this file ?

Comment (ChetanSharma-msft) [reply samples/graph-change-notification-team-channel-nodejs/nodejs/client/src/models/dashboard-state.js:5]: If we are using it, rename it properly

File: samples/graph-change-notification-team-channel-nodejs/nodejs/client/src/style/card.css [hunk 14]
```diff
@@ -0,0 +1,35 @@
...
+    margin-left: 15px;
+    color: white;
+    font-weight: bold;
+    float: right;
+    font-size: 22px;
+    line-height: 20px;
+    cursor: pointer;
+    transition: 0.3s;
+  }
+
```
Comment (ChetanSharma-msft) [inline samples/graph-change-notification-team-channel-nodejs/nodejs/client/src/style/card.css:32]: Remove if we are not using all of these CSS

File: samples/graph-change-notification-team-channel-nodejs/nodejs/client/src/style/style.css [hunk 15]
```diff
@@ -0,0 +1,30 @@
...
+.manage-icons:hover {
+    color: #5b5fc7;
+    background-color: "#5B00C7";
+}
+
+.text-fields-label {
+    width: 130px;
+}
+
+.input-fields-label {
```
Comment (ChetanSharma-msft) [inline samples/graph-change-notification-team-channel-nodejs/nodejs/client/src/style/style.css:25]: Remove if we are not using all of these CSS

File: samples/graph-change-notification-team-channel-nodejs/nodejs/package.json [hunk 16]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "graph-notification",
+  "version": "1.0.0",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node server.js"
```
Comment (ChetanSharma-msft) [inline samples/graph-change-notification-team-channel-nodejs/nodejs/package.json:7]: Why do we have 2 package.json and 2 package-lock.json ??

Remove one which is not in use.

File: samples/graph-change-notification-team-channel-nodejs/nodejs/server.js [hunk 17]
```diff
@@ -0,0 +1,66 @@
...
+    res.send();
+  }
+}
+
+// Listen for incoming requests.
+app.post('/api/notifications', notification);
+
+app.listen(3000, function () {
+  console.log('app listening on port 3000!');
+});
```
Comment (ChetanSharma-msft) [inline samples/graph-change-notification-team-channel-nodejs/nodejs/server.js:66]: Is this port number correct ?
Generally we are mentioning 3978 around.
Refer any other sample to clarify it.

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added my comments.

File: samples/graph-change-notification-team-channel-nodejs/nodejs/Assests/sample.json [hunk 18]
```diff
@@ -0,0 +1,68 @@
...
+        "value": "OfficeDev"
+      },
+      {
+        "key": "TEAMS-SERVER-LANGUAGE",
+        "value": "csharp"
+      },
+      {
+        "key": "TEAMS-SERVER-PLATFORM",
+        "value": "netframework"
+      },
```
Comment (ChetanSharma-msft) [inline samples/graph-change-notification-team-channel-nodejs/nodejs/Assests/sample.json:28]: This is not a csharp sample, kindly update these as per NodeJS sample.

File: samples/graph-change-notification-team-channel-nodejs/nodejs/Assests/sample.json [hunk 19]
```diff
@@ -0,0 +1,68 @@
...
+      {
+        "key": "TEAMS-FEATURES",
+        "value": "tab"
+      }
+    ],
+    "thumbnails": [
+      {
+        "type": "image",
+        "order": 100,
+        "url": "https://raw.githubusercontent.com/OfficeDev/Microsoft-Teams-Samples/v-mfurquan/graph-change-notification-team-channel-nodejs/samples/graph-change-notification-team-channel-nodejs/nodejs/Images/ChangeNotifications.gif",
```
Comment (ChetanSharma-msft) [inline samples/graph-change-notification-team-channel-nodejs/nodejs/Assests/sample.json:38]: This url should not contain the path of your branch "v-mfurquan"
It should be main branch.

File: samples/graph-change-notification-team-channel-nodejs/nodejs/Assests/sample.json [hunk 20]
```diff
@@ -0,0 +1,68 @@
...
+        "alt": "Solution UX showing Graph Change Notification Team/Channel nodejs"
```
Comment (ChetanSharma-msft) [inline samples/graph-change-notification-team-channel-nodejs/nodejs/Assests/sample.json:39]: Update it as "Graph Change Notification For Team/Channel using NodeJs"

File: samples/graph-change-notification-team-channel-nodejs/nodejs/.env [hunk 21]
```diff
@@ -0,0 +1,10 @@
+MicrosoftAppId = "<<MicrosoftAppId>>"
+MicrosoftAppPassword = "<<MicrosoftAppPassword>>"
+MicrosoftAppTenantId = "<<MicrosoftAppTenantId>>"
+AllowedHosts = "*"
+notificationUrl= "<<ngrok>>/api/notifications"
+BaseUrl= "<<BaseUrl>>"
+Base64EncodedCertificate= "<<Base64EncodedCertificate>>"
+EncryptionCertificateId= "meeting-notification"
+CertificateThumbprint= "<<Certificate Thumbprint>>",
+PRIVATE_KEY_PATH=<<Keypath>>
```
Comment (ChetanSharma-msft) [inline samples/graph-change-notification-team-channel-nodejs/nodejs/.env:10]: Check if we have added description or how to get this value in ReadMe file.

File: samples/graph-change-notification-team-channel-nodejs/nodejs/README.md [hunk 22]
```diff
@@ -0,0 +1,118 @@
...
+    ```
+
+  - Launch Visual Studio code
+  - File -> Open Folder
+  - Navigate to `samples/graph-change-notification-team-channel/nodejs` folder.
+
+  - Update the `.env` file
+
+   Update configuration with the ```MicrosoftAppId```, ```MicrosoftAppPassword```, ```MicrosoftAppTenantId```,
+   ```Base64EncodedCertificate```, ```EncryptionCertificateId```, ```PRIVATE_KEY_PATH```
```
Comment (ChetanSharma-msft) [inline samples/graph-change-notification-team-channel-nodejs/nodejs/README.md:83]: Mention the steps, how to get these values.

```Base64EncodedCertificate```, ```EncryptionCertificateId```, ```PRIVATE_KEY_PATH```

File: samples/graph-change-notification-team-channel-nodejs/nodejs/README.md [hunk 23]
```diff
@@ -0,0 +1,118 @@
...
+   ``` 
+
+ - Run ngrok - point to port 3978
+
+   ```bash
+     ngrok http -host-header=rewrite 3978
+   ```  
+
+## Instruction for .env
+1. Provide MicrosoftAppId, MicrosoftAppPassword and MicrosoftAppTenantId in the .env that is created in Azure.
```
Comment (ChetanSharma-msft) [inline samples/graph-change-notification-team-channel-nodejs/nodejs/README.md:106]: Provide MicrosoftAppId, MicrosoftAppPassword and MicrosoftAppTenantId in the .env that is created in previous step while doing App Registration.

File: samples/graph-change-notification-team-channel-nodejs/nodejs/README.md [hunk 24]
```diff
@@ -0,0 +1,118 @@
...
+     npm start
+   ``` 
+
+ - Run ngrok - point to port 3978
+
+   ```bash
+     ngrok http -host-header=rewrite 3978
+   ```  
+
+## Instruction for .env
```
Comment (ChetanSharma-msft) [inline samples/graph-change-notification-team-channel-nodejs/nodejs/README.md:105]: I think, this steps should come along with the this step
Update configuration with the ```MicrosoftAppId```, ```MicrosoftAppPassword```, ```MicrosoftAppTenantId```,
   ```Base64EncodedCertificate```, ```EncryptionCertificateId```, ```PRIVATE_KEY_PATH```

File: samples/graph-change-notification-team-channel-nodejs/nodejs/README.md [hunk 25]
```diff
@@ -0,0 +1,118 @@
...
+1. Provide MicrosoftAppId, MicrosoftAppPassword and MicrosoftAppTenantId in the .env that is created in Azure.
+2. Provide the ngrok url as  "BaseUrl" in appsetting on which application is running on.
+3. You should be having Base64EncodedCertificate and CertificateThumbprint value from *Create and install Self-Signed certificate* step.
+4. Use Certificate in "PEM" format and add path in this way "PRIVATE_KEY_PATH=PathName.pem" in .env file.
+
+## Instruction for manifest
+1. Fill any GUID for <APP-ID>. You can also put your MicrosoftAppId here.
```
Comment (ChetanSharma-msft) [inline samples/graph-change-notification-team-channel-nodejs/nodejs/README.md:112]: You haven't updated it.

File: samples/graph-change-notification-team-channel-nodejs/nodejs/client/src/components/channelnotification.jsx [hunk 26]
```diff
@@ -0,0 +1,109 @@
...
+            microsoftTeams.app.getContext().then((context) => {
+                this.setState({ teamId: context.team.groupId });
+                this.initializeData(context.team.groupId);
+            })
+        });
+    }
+
+    ///Summary///
+    ///Passing {teamId} to team controller for change notification subscription.
+    ///Summary///
```
Comment (ChetanSharma-msft) [inline samples/graph-change-notification-team-channel-nodejs/nodejs/client/src/components/channelnotification.jsx:34]: Is this the correct way of providing summary ?

File: samples/graph-change-notification-team-channel-nodejs/nodejs/server.js [hunk 27]
```diff
@@ -0,0 +1,52 @@
...
+            res.status(500).send();
+        }
+    }
+}
+
+// Listen for incoming requests.
+app.post('/api/notifications', notification);
+
+app.listen(3000, function () {
+    console.log('app listening on port 3000!');
```
Comment (ChetanSharma-msft) [inline samples/graph-change-notification-team-channel-nodejs/nodejs/server.js:51]: No updates on this ?

File: samples/graph-change-notification-team-channel-nodejs/nodejs/README.md [hunk 28]
```diff
@@ -0,0 +1,118 @@
+---
+page_type: sample
```
Comment (ChetanSharma-msft) [inline samples/graph-change-notification-team-channel-nodejs/nodejs/README.md:2]: Make sure that you have linked this project with main readme file which is present at the root folder.

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added my review comments

File: samples/graph-change-notification-team-channel-nodejs/nodejs/Assests/sample.json [hunk 29]
```diff
@@ -0,0 +1,68 @@
...
+        "key": "TEAMS-FEATURES",
+        "value": "tab"
+      }
+    ],
+    "thumbnails": [
+      {
+        "type": "image",
+        "order": 100,
+        "url": "https://raw.githubusercontent.com/OfficeDev/Microsoft-Teams-Samples/graph-change-notification-team-channel-nodejs/samples/graph-change-notification-team-channel-nodejs/nodejs/Images/ChangeNotifications.gif",
+        "alt": "Graph Change Notification Team/Channel using Nodejs"
```
Comment (ChetanSharma-msft) [inline samples/graph-change-notification-team-channel-nodejs/nodejs/Assests/sample.json:39]: Graph Change Notification for Team/Channel using NodeJS

File: samples/graph-change-notification-team-channel-nodejs/nodejs/helper/certHelper.js [hunk 30]
```diff
@@ -0,0 +1,129 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+
```
Comment (ChetanSharma-msft) [inline samples/graph-change-notification-team-channel-nodejs/nodejs/helper/certHelper.js:4]: Remove empty extra link

File: samples/graph-change-notification-team-channel-nodejs/nodejs/helper/certHelper.js [hunk 31]
```diff
@@ -0,0 +1,129 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
```
Comment (ChetanSharma-msft) [inline samples/graph-change-notification-team-channel-nodejs/nodejs/helper/certHelper.js:2]: Also this file name should be like "certificate-helper.js"

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added my comments.

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added comments

File: samples/graph-change-notification-team-channel-nodejs/nodejs/.env (hunk 21 above)
Comment (Prithvi-MSFT) [inline samples/graph-change-notification-team-channel-nodejs/nodejs/.env:10]: Changes this to PRIVATE_KEY_PATH=<<PrivateKeyFileName>>

File: samples/graph-change-notification-team-channel-nodejs/nodejs/README.md [hunk 32]
```diff
@@ -0,0 +1,118 @@
...
+To include resource data of graph notifications, this Graph API require self-signed certificate. Follow the below steps to create and manage certificate.
+
+1. You can self-sign the certificate, since Microsoft Graph does not verify the certificate issuer, and uses the public key for only encryption.
+
+2. Use [Azure Key Vault](https://docs.microsoft.com/en-us/azure/key-vault/key-vault-whatis) as the solution to create, rotate, and securely manage certificates. Make sure the keys satisfy the following criteria:
+
+    - The key must be of type `RSA`
+    - The key size must be between 2048 and 4096 bits
+
+3. Follow this documentation for the steps - [**Create and install Self-Signed certificate**](Certificate Documentation/README.md)
```
Comment (Prithvi-MSFT) [inline samples/graph-change-notification-team-channel-nodejs/nodejs/README.md:66]: This link is broken

File: samples/graph-change-notification-team-channel-nodejs/nodejs/README.md [hunk 33]
```diff
@@ -0,0 +1,118 @@
...
+
+  - Update the `.env` file
+
+   Update configuration with the ```MicrosoftAppId```, ```MicrosoftAppPassword```, ```MicrosoftAppTenantId```,
+   ```Base64EncodedCertificate```, ```EncryptionCertificateId```, ```PRIVATE_KEY_PATH```
+
+### Instruction for .env
+1. Provide MicrosoftAppId, MicrosoftAppPassword and MicrosoftAppTenantId in the .env that is created in Azure.
+2. Provide the ngrok url as  "BaseUrl" in appsetting on which application is running on.
+3. You should be having Base64EncodedCertificate and CertificateThumbprint value from *Create and install Self-Signed certificate* step.
```
Comment (Prithvi-MSFT) [inline samples/graph-change-notification-team-channel-nodejs/nodejs/README.md:88]: Thumbprint value is not required you can remove that

File: samples/graph-change-notification-team-channel-nodejs/nodejs/README.md [hunk 34]
```diff
@@ -0,0 +1,118 @@
...
+4. Use Certificate "PEM" format and add path in this way "PRIVATE_KEY_PATH=PathName.pem" in .env file.
```
Comment (Prithvi-MSFT) [inline samples/graph-change-notification-team-channel-nodejs/nodejs/README.md:89]: Refer this sample and add similar steps for configuring certificates
https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/v-pritka/meeting-notification-nodejs/samples/graph-meeting-notification/nodejs

File: samples/graph-change-notification-team-channel-nodejs/nodejs/.env [hunk 35]
```diff
@@ -0,0 +1,10 @@
+MicrosoftAppId = "<<MicrosoftAppId>>"
+MicrosoftAppPassword = "<<MicrosoftAppPassword>>"
+MicrosoftAppTenantId = "<<MicrosoftAppTenantId>>"
+AllowedHosts = "*"
+notificationUrl= "<<ngrok>>/api/notifications"
+BaseUrl= "<<BaseUrl>>"
+Base64EncodedCertificate= "<<Base64EncodedCertificate>>"
+EncryptionCertificateId= "meeting-notification"
+CertificateThumbprint= "<<Certificate Thumbprint>>",
```
Comment (Prithvi-MSFT) [inline samples/graph-change-notification-team-channel-nodejs/nodejs/.env:9]: Is thumbprint required ? Remove if not needed

**Review by Prithvi-MSFT — COMMENTED** [review]
Added my comments

**Comment by Prithvi-MSFT** [comment]
Change the project folder name from **graph-change-notification-team-channel-nodejs** to **graph-change-notification-team-channel**

**Review by Prithvi-MSFT — APPROVED** [review]

File: samples/graph-change-notification-team-channel/nodejs/Assests/sample.json [hunk 36]
```diff
@@ -0,0 +1,68 @@
...
+  {
+    "name": "officedev-microsoft-teams-samples-graph-change-notification-team-channel",
+    "source": "officeDev",
+    "title": "Graph Change Notification Team/Channel",
+    "shortDescription": "Graph change notification for team and channel",
+    "url": "https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/graph-change-notification-team-channel/nodejs",
+    "longDescription": [
+      "This sample demonstrates the use of Team/Channel subscription that will post notifications when user Create/Edit/Delete the team/channel."
+    ],
+    "creationDateTime": "2022-09-19",
```
Comment (Wajeed-msft) [inline samples/graph-change-notification-team-channel/nodejs/Assests/sample.json:11]: Change to latest date.

**Review by Wajeed-msft — APPROVED** [review]

File: samples/graph-change-notification-team-channel/nodejs/README.md [hunk 37]
```diff
@@ -0,0 +1,118 @@
...
+description: This is a sample application which demonstrates the use of team/Channel subscription that will post notifications when user create/edit/delete team/channel.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- nodejs
+extensions: 
+ contentType: samples
+ createdDate: "19-09-2022 01:30:15"
```
Comment (Wajeed-msft) [inline samples/graph-change-notification-team-channel/nodejs/README.md:12]:
```suggestion
 createdDate: "10-03-2022 11:30:15 AM"
```

**Pull request merged**